contractcourt/commit_sweep_resolver.go

```diff
@@ -298,15 +311,6 @@ func newCommitSweepResolverFromReader(r io.Reader, resCfg ResolverConfig) (
 	return c, nil
 }
 
-// AttachConfig should be called once a resolved is successfully decoded from
```

Comment:
nit: delete in the same commit as we remove the last reference to this method.

Reply:
This is the last commit that contains the last ref?